docs/connect/mongo-compass.mdx

```diff
@@ -77,7 +77,7 @@ Here is how to connect MongoDB Compass to MindsDB using either MindsDB Cloud or
     Here is what you need to connect:
     
     ```
-    Host: <dedicated instace ip>:<port>
+    Host: <dedicated instace ip>:3306
```

Comment:
Please correct `instace` to `instance` (as it was mentioned in the issue as well.